dbld/images/bionic.dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, "
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN=true
+ENV LANG C.UTF-8
+ENV OS_PLATFORM bionic
+
+RUN apt-get update -qq -o Acquire::CompressionTypes::Order::=gz
+
+COPY required-apt/packages.txt /packages.txt
+RUN grep ${OS_PLATFORM} /packages.txt | cut -d"[" -f1 | xargs apt-get install --no-install-recommends --yes
+
```

Comment:
this grep and cut magic should be a helper script that invokes apt based on the packages file.

Reply:
Ok, agreed. In the next update I will fix this.

Reply:
During the development of dbld, I have intentionally tried to avoid extracting this command from the Dockerfile. IMHO "hiding" the install steps somehow mystifies the actual work. (Using text files was already a big step back.)

i.e.: Since the Dockerfile's syntax is limited, the easiest Dockerfile will be:
`FROM imagename`
`COPY bar /foo`
`RUN foo/create.sh`
`ENTRYPOINT["foo/foo.sh"]`

Obviously, we don't want to do that. However, IF we take this step, I think we can use some other formats which will furtherly reduce the maintenance complexity of the package lists, and provide extra functionality at the same time. i.e.:
- warning about duplications (some kind of stylecheck)
- creating "groups" of platforms, to sorten the lists.
- able to provide version numbers (the same method can be used for pip)